packages/catlog/src/dbl/model_morphism.rs

```diff
@@ -333,14 +334,12 @@ mod tests {
 
         let maps = DiscreteDblModelMapping::morphisms(&positive_loop, &positive_loop).find_all();
         assert_eq!(maps.len(), 2);
-        assert!(matches!(maps[0].apply_mor(&pos), Some(Path::Id(_))));
-        assert!(matches!(maps[1].apply_mor(&pos), Some(Path::Seq(_))));
```

Comment:
OK, I know this test was bad, but this feels like throwing out the baby with the bathwater! I'm tempted to add an `Ord` implementation to `Path` (whenever `V` and `E` are `Ord`) consistent with the [lexicographic order](https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Ord-for-Vec%3CT,+A%3E) for `Vec`.

For now, can you just comment these out but not delete them? I'll fix them later.

Reply:
I think that this actually makes sense: if `maps.len() == 2`, and `x` and `y` are both in `map_obs`, then that's all we care about...

Reply:
Oh, I thought this was for another one. I'll do the same strategy here.